modules/s3/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  terraform_required_version = "~> 1.3"
```

Comment:
This is a Terragrunt parameter. For TF its just `required_version`

Reply:
yep, saw it